Source/TextKit/ASTextKitCoreTextAdditions.m

```diff
@@ -236,6 +237,7 @@ + (instancetype)paragraphStyleWithCTParagraphStyle:(CTParagraphStyleRef)coreText
     if (CTParagraphStyleGetValueForSpecifier(coreTextParagraphStyle, kCTParagraphStyleSpecifierLineSpacing, sizeof(lineSpacing), &lineSpacing))
```

Comment:
Need to add back support for line spacing on tvOS since `kCTParagraphStyleSpecifierLineSpacing` is unavailable on tvOS.